src/extra/fromEvent.ts

```diff
@@ -134,12 +134,18 @@ function isEmitter(element: any): boolean {
  * dispatched to any EventTarget beneath it in the DOM tree. Defaults to false.
  * @return {Stream}
  */
-export default function fromEvent(element: EventTarget | EventEmitter,
-                                  eventName: string,
-                                  useCapture: boolean = false): Stream<Event|any> {
+
+function fromEvent<T>(element: EventEmitter, eventName: string): Stream<T>;
```

Comment:
Could you use `<T = any>`?

Reply:
Maybe it's not necessary since you can already just call the function without specifying the generic and TypeScript will figure it out for you.
But I can add it either way.

Reply:
When TypeScript figures it out, then it ignores the `= any`, when it doesn't figure it out, it uses `any` instead of `{}`, which I think is better since `{}` is rarely useful, and the current `fromEvent` outputs `Stream<Event|any>`

Reply:
Done!